cilium/cmd/bpf_lb_maglev_get.go

```diff
@@ -47,24 +46,23 @@ var bpfMaglevGetCmd = &cobra.Command{
 		lookupTables := map[string][]string{}
 		found := false
 
-		for _, name := range []string{lbmap.MaglevOuter4MapName, lbmap.MaglevOuter6MapName} {
-			// See bpf_lb_maglev_list.go comment for why we use low-level routines
-			// to open and access the maps
-			if m, err := bpf.OpenMap(name); err != nil {
-				continue
-			} else {
-				err := bpf.LookupElement(m.GetFd(), key.GetKeyPtr(), val.GetValuePtr())
-				if err != nil {
-					if errors.Is(err, unix.ENOENT) {
-						continue
-					}
-					Fatalf("Unable to retrieve entry from %s with key %s: %s",
-						name, key, err)
-				} else {
-					found = true
-					parseMaglevEntry(key, val, lookupTables)
+		tableSize, err := lbmap.MaybeInitMaglevMapsByProbingTableSize()
```

Comment:
Nit: this function name seems unnecessarily long. Couldn't it just be `MaybeInitMaglevMaps()`? The godoc on the function can described how it's doing it, without it being in the function name itself